README.md

```diff
@@ -530,17 +530,19 @@ with the `mod_auth_gssapi` module.
 
 ## Nexus
 
-### Nexus For npm
+### Nexus For Java Script
```

Comment:
I decided to have one section for both yarn and npm because yarn is just a CNAME for js nexus repos.